aiohttp/web_fileresponse.py

```diff
@@ -179,31 +184,72 @@ def _get_file_path_stat_encoding(
                 # Do not follow symlinks and ignore any non-regular files.
                 st = compressed_path.lstat()
                 if S_ISREG(st.st_mode):
-                    return compressed_path, st, file_encoding
+                    fobj = compressed_path.open("rb")
+                    with suppress(OSError):
```

Comment:
What's being suppressed here that isn't caught with the other `suppress`?

Reply:
`fstat() may not be available on all platforms`

Reply:
Got it.  I thought this was under the previous `with`, which I now realize is not the case.